.meta-updater/src/index.ts

```diff
@@ -105,16 +105,75 @@ async function updateTSConfig (
       // This is to avoid a circular graph (which TypeScript references do not support.
       continue
     }
-    references.push({ path: relativePath })
+    linkValues.push(relativePath)
   }
-  await writeJsonFile(path.join(dir, 'tsconfig.lint.json'), {
-    extends: './tsconfig.json',
-    include: [
-      'src/**/*.ts',
-      'test/**/*.ts',
-      '../../__typings__/**/*.d.ts',
-    ],
-  }, { indent: 2 })
+  linkValues.sort((a, b) => a.localeCompare(b))
```

Comment:
Nit, but you probably just want to call sort; these are already strings and locale compare is not a good comparison for anything but human readable strings since it will differ on different people's machines based on their language setting.

Reply:
The `localeCompare` existed before this PR, but only looks new since I moved the side-effectful sort onto its own line. Would be good to fix regardless.

Reply:
> Nit, but you probably just want to call sort without a comparison function; these are already strings and locale compare is not a good comparison for anything but human readable strings since it will differ on different people's machines based on their language setting. I've hit this too many times before for code gen.

I've seen `localeCompare` in this codebase many times before, I'm not sure if they are appropriate though.

Reply:
Our linting forces the usage of a function in `.sort()` as `.sort()` can behave unexpectedly on numbers. It will do text sort even for an array with numbers.

Reply:
How about this?

```typescript
a.localeCompare(b, 'en-US')
```

This should make the comparison deterministic and independent on system locale.